src/pages/HomePage/ChangeView/ChangeView.jsx

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable import/no-absolute-path */
+/* eslint-disable import/extensions */
+/* eslint-disable import/no-unresolved */
+import MapIcon from '@mui/icons-material/Map';
+import SearchIcon from '@mui/icons-material/Search';
+import { Box, Button, ButtonGroup } from '@mui/material';
+import { styled } from '@mui/material/styles';
+
+import theme from '/src/theme/theme';
+
+import styles from './ChangeView.module.scss';
+
+const ChangerButton = styled(Button)({
+  flexDirection: 'column',
+  width: '83px',
+  height: '56px',
+  backgroundColor: theme.palette.primary.main,
+  borderRadius: '0px',
+  borderRight: 'none',
+  '&:hover': {
+    backgroundColor: 'none !important',
+  },
+  '&:focus': {
+    backgroundColor: 'none',
+  },
+});
+
+function ChangeView(props) {
```

Comment:
`const state = { ...props }` doesn't make much difference. You could unpack `props` here (`function ChangeView({isSetToggle, isToggle }) {`) and you don't need to worry about eslint warnings as they should be solved once TypeScript is introduced.